app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::ReviewsController < ApplicationController
+    def index
+        render json: Review.all
+    end
+end
```

Comment:
similar to Christie's question on `routes.rb` about whether we need a dedicated "index" path for reviews, do we actually need an "index" method here, or even a "ReviewsController" overall, since we're pulling reviews into the podcast show page as part of the "podcast" object returned via the fetch request to "/api/v1/podcasts/${id}"